Source/Game/Game/Editor/AssetBrowser.cpp

```diff
@@ -0,0 +1,490 @@
+#include "AssetBrowser.h"
+
+#include <Base/Util/DebugHandler.h>
+#include <Game/Util/ImGui/FakeScrollingArea.h>
+
+namespace Editor
+{
+    AssetBrowser::AssetBrowser()
+        : BaseEditor(GetName(), true),
+          _tree(nullptr), _currentNode(nullptr)
```

Comment:
We generally put one of the constructor initializers per line:
: BaseEditor()
, _tree(nullptr)
, currentNode(nullptr)